lib/modules/manager/velaci/extract.ts

```diff
@@ -0,0 +1,56 @@
+import { load } from 'js-yaml';
+import { logger } from '../../../logger';
+import { getDep } from '../dockerfile/extract';
+import type { PackageDependency, PackageFile } from '../types';
+import type { VelaPipelineConfiguration } from './types';
+
+export function extractPackageFile(file: string): PackageFile | null {
+  let doc: VelaPipelineConfiguration | undefined;
+
+  try {
+    doc = load(file, { json: true }) as VelaPipelineConfiguration;
+  } catch (err) {
+    logger.warn({ err, file }, 'Failed to parse Vela file.');
+    return null;
+  }
+
+  const deps: PackageDependency[] = [];
+
+  // iterate over steps
+  for (const step of doc.steps ?? []) {
+    const dep: PackageDependency = getDep(step.image);
```

Comment:
```suggestion
    const dep = getDep(step.image);
```
I think we don't need the explicit type here.